aggregation/simple/src/main/java/com/spotify/heroic/aggregation/simple/PointsBelowInstance.java

```diff
@@ -21,11 +21,22 @@
 
 package com.spotify.heroic.aggregation.simple;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
 import java.beans.ConstructorProperties;
 
+@Data
+@EqualsAndHashCode(callSuper = true)
 public class PointsBelowInstance extends MetricMappingAggregation {
+    private final double threshold;
+
+    @JsonCreator
```

Comment:
`@ConstructorProperties` completely replaces `@JsonCreator`. Therefore `@JsonCreator` and `@JsonProperty` annotations should not be present if `@ConstructorProperties` is present.

Apologies if I didn't make that clear enough.

Reply:
I'm pretty sure that's what fixed this issue in the first place? Hang on, going to test it now without JSONCreatory/Property.

Reply:
Ok, all set.